epcis/src/main/java/io/openepcis/model/epcis/modifier/CustomExtensionAdapter.java

```diff
@@ -26,7 +26,7 @@
 
 public class CustomExtensionAdapter extends XmlAdapter<MapWrapper, Map<String, Object>> {
 
```

Comment:
context is shared - please fix, each thread / request must be running on it's own context